examples/lib/stories/components/game_in_game_example.dart

```diff
@@ -36,9 +36,10 @@ class GameChangeTimer extends TimerComponent
   @override
   void onTick() {
     final child = gameRef.draggablesGame.square;
-    final newParent = child.parent == gameRef.draggablesGame
-        ? gameRef.composedGame.parentSquare
-        : gameRef.draggablesGame;
-    child.changeParent(newParent);
+    if (child.parent == gameRef.draggablesGame) {
+      child.parent = gameRef.composedGame.parentSquare;
+    } else {
+      child.parent = gameRef.draggablesGame;
+    }
```

Comment:
is this related? we shouldn't use the method changeParent?

Reply:
There was something weird going on with this snippet: my Dart decided that the common superclass between these RHSs is `Object`. The `changeParent` method can still be used, I reverted this change to almost the same as it was before.

Reply:
sg, I was just curious if it was being removed or something